app/jobs/reports/daily_auths_report.rb

```diff
@@ -17,10 +17,22 @@ def perform(report_date)
 
       _latest, path = generate_s3_paths(REPORT_NAME, 'json', now: report_date)
 
-      upload_file_to_s3_bucket(
-        path: path,
-        body: report_body.to_json,
-        content_type: 'application/json',
+      [
+        gen_s3_bucket_name, # default reporting bucket
+        public_bucket_name,
+      ].each do |bucket_name|
+        upload_file_to_s3_bucket(
+          path: path,
+          body: report_body.to_json,
+          content_type: 'application/json',
+          bucket_name: bucket_name,
+        )
+      end
+    end
+
+    def public_bucket_name
```

Comment:
Not sure if we plan to use it for anything else, but would it make sense to define this in the base class?

Also wondering about how we have a "gen(erate)" method for the standard bucket, but not for this. Is `Identity::Hostdata.bucket_name` costly? I don't have much an opinion one way or the other whether we memoize or not, just noticed it as a small inconsistency.

Reply:
it's super cheap to build those strings... IMO not worth memoizing. I had some changes where I got rid of the `gen_` method entirely and just had `bucket`.... but didn't want change too much. Now that you bring it up, I'll bring it back

Reply:
see 7e488b24f064ea7670a76d0cfa569b1675abfc54